admin/fluent-bit/Makefile

```diff
@@ -0,0 +1,55 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=fluent-bit
+PKG_VERSION:=3.1.0
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/fluent/fluent-bit.git
+PKG_SOURCE=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_VERSION=v$(PKG_VERSION)
```

Comment:
Here you are mixing at least it looks like two things. Checking git sources and downloading tarball. You should decide which one would you want to use.

Reply:
yes, I realise, updated, thanks!